Sandbox/src/Game/Interface/GearInterface.cpp

```diff
@@ -1,17 +1,83 @@
 #include "GearInterface.h"
 
 GearInterface::GearInterface(Gear& gear)
-	: IMenuTab("Graphics/panel_brown.png", 32, false, false), mGear(gear)
+	: IMenuTab("Graphics/panel_gear.png", 32, false, false), mGear(gear)
 {
-	for (int y = 0; y < 5; y++)
+	InitializeGearPositions();
+}
+
+void GearInterface::InitializeGearPositions()
+{
+	mGearPositions = {
```

Comment:
You don't need to create a new container to store gear positions. You can just use mSlotPositions because they would be  identical.

Reply:
nice yea the use of mIcons and mSlotPos makes this cleaner!